test/libponyc/badpony.cc

```diff
@@ -426,3 +426,16 @@ TEST_F(BadPonyTest, AnnotatedIfClause)
 
   TEST_COMPILE(src);
 }
+
+TEST_F(BadPonyTest, CapSubtypeInConstrainSubtyping)
+{
+  // From PR #1816
+  const char* src =
+    "trait T\n"
+    "  fun alias[X: Any iso](x: X!) : X^\n"
+    "class C is T\n"
+    "  fun alias[X: Any tag](x: X!) : X^ => x\n";
```

Comment:
It's a small style point, but could you remove the "extra" space before the colon that precedes the return type?

This would make it more closely match the prevailing style in these tests and in the standard libraries.